docs/proposals/fluentd-limit-requests-to-apiserver.md

```diff
@@ -0,0 +1,63 @@
+# Modifications to Fluentd Metadata Retrieval Proposal
```

Comment:
Can you add a clear problem statement like: "Current design of fluentd pods maintain open connections to the API server which limit the number of nodes because API server has a cap on the number of open connections.  The fluentd pods currently need the connection to the API server so that they can query for additional metadata to enrich the logs with labels, annotations, host, and pod ID."